proto/sentry_protos/snuba/v1/endpoint_get_traces.proto

```diff
@@ -0,0 +1,76 @@
+syntax = "proto3";
+
+package sentry_protos.snuba.v1;
+
+import "sentry_protos/snuba/v1/request_common.proto";
+import "sentry_protos/snuba/v1/trace_item_attribute.proto";
+import "sentry_protos/snuba/v1/trace_item_filter.proto";
+
+// TraceAttribute represents the attribute of a trace.
+message TraceAttribute {
+  // Key lists the available trace attribute keys you can query.
+  enum Key {
+    KEY_UNSPECIFIED = 0;
+    KEY_TRACE_ID = 1;
+    // KEY_START_TIMESTAMP will return the earliest timestamp seen in the trace.
+    KEY_START_TIMESTAMP = 2;
+    // KEY_ROOT_SPAN_NAME will return the name of the root (segment) span of
+    // the trace.
+    KEY_ROOT_SPAN_NAME = 3;
```

Comment:
Will this have the heuristics we spoke about built in or will there be other attributes for those cases?

Reply:
This won't have the heuristic of finding either the root span name or the first http span name. I think we should add another attribute key for that and you can choose which one you want.